### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by wouterzwerink**
Small typo I ran into when calling set_metric_names incorrectly..

### CODE REVIEW

File: optuna/study/study.py [hunk 1]
```diff
@@ -1007,7 +1007,7 @@ def objective(trial):
         """
         if len(self.directions) != len(metric_names):
             raise ValueError(
-                "The number of objectives must match thhe length of the metric names."
+                "The number of objectives must match the length of the metric names."
```
Comment (toshihikoyanase) [inline optuna/study/study.py:1010]: @wouterzwerink Thank you for your PR. The change seems obvious, but the CI failed. Could you follow the reviewdog's comment, please?

Comment (wouterzwerink) [reply optuna/study/study.py]: Ha, seems like it fits within the line limit after removing this one character

Comment (toshihikoyanase) [reply optuna/study/study.py]: Absolutely. Thanks for the update.

**Review by toshihikoyanase — APPROVED** [review]
LGTM

**Comment by toshihikoyanase** [comment]
I merged this PR since it contains only a small fix in an error message.

**Pull request merged**